tests/conftest.py

```diff
@@ -197,3 +197,8 @@ def cleanup():
     email = "lektortest@example.com"
     subprocess.check_call(['git', 'config', 'user.email', email])
     return email
+
+
+@pytest.fixture(scope='session')
+def splinter_screenshot_dir(request):
+    return os.path.join(os.path.abspath(request.config.option.splinter_screenshot_dir), 'tmp')
```

Comment:
Can you explain what this splinter screenshot directory is for? Are you expecting that the developer running the tests will look in this directory and view the screenshots after the tests run? If that's the case, you should use a more descriptive name than "tmp" for this directory, and you should add some documentation to the `README` file about viewing screenshots after running tests.

If you don't expect people to view screenshots after every run, maybe this fixture isn't necessary? Or maybe you can use the [`tmpdir` fixture](http://docs.pytest.org/en/latest/tmpdir.html) to easily get a temporary directory? That way, you won't have to add this directory to the `.gitignore` file.